### ISSUE TITLE
Event listener breaks on multi-line, tab indented element

### ISSUE THREAD

**Issue by riovir** [hybridsjs/hybrids#97]
# Version
4.1.4

# Reproduction

Modify [/test/spec/html.js](https://github.com/hybridsjs/hybrids/blob/master/test/spec/html.js#L241-L243) to use a multiline style, tab indented `onclick` listener, then run the tests. 

```js
describe('event attribute expression', () => {
  /* eslint-disable no-tabs */
  const render = (value) => html`<button
		onclick=${value}
  ></button><button onclick=${value}></button>`;
  /* eslint-enable no-tabs */
  const renderWithQuotes = (value) => html`<button onclick="${value}"></button>`;
```

(This issue is behind [the odd line](https://gitlab.com/riovir/demo-components/-/blob/5d0c9cbc3bdd233e68ee855348705dd3e0ca410e/src/components/b-navbar/b-navbar.js#L20) in this sandbox component pack.)

# What is expected?
The tests to pass just as if I have been using space indentation.

# What is actually happening?
The source code of the event handler gets inlined as an attribute in the rendered element.

**Comment by smalluban** [comment]
Fixed in `v4.1.5`. Thanks for creating an issue.

**Comment by riovir** [comment]
Ahoy! Sure thing. Impressive API, docs, and reaction time there! Cheers!
